packages/components/src/components/loaders/ProgressBar/ProgressBar.tsx

```diff
@@ -1,35 +1,58 @@
-import styled from 'styled-components';
+import styled, { useTheme } from 'styled-components';
 
-const Wrapper = styled.div`
-    background: ${({ theme }) => theme.backgroundNeutralSubdued};
+import { CSSColor } from '@trezor/theme';
+
+const Wrapper = styled.div<{ $color: CSSColor }>`
+    background: ${({ $color }) => $color};
     width: 100%;
     overflow: hidden;
 `;
 
 type ValueProps = {
     $max: number;
     $value: number;
+    $color: CSSColor;
 };
 
 const Value = styled.div<ValueProps>`
-    background: ${({ theme }) => theme.iconPrimaryDefault};
+    background: ${({ $color }) => $color};
     height: 5px;
     max-width: 100%;
-    width: 1%;
-    transform: ${({ $max, $value }) => `scaleX(${(100 / $max) * $value})`};
```

Comment:
There is a rounding bug, so I rewrote it with width. Looks like everything is still working.

Reply:
Can't we just use Math.round?:)